### DISCUSSION TITLE
Rounded border corners

### DISCUSSION THREAD

**Discussion by JaiTheArtist** [sayanarijit/xplr#447] [Q&A]
I was curious if anyone figured out how to change the tui border to have rounded corners.
I see the defaults in the Panel UI of the config , but no way of changing the actual style of them. 
`xplr.config.general.panel_ui.default.borders = {
  "Top",
  "Right",
  "Bottom",
  "Left",
}`

**Comment by sayanarijit** [comment]
Not yet implemented. The underlying library tui-rs [supports rounded borders](https://github.com/fdehau/tui-rs/blob/6069d89dee8e43b99c3938c2f747c5274ce4aef0/examples/block.rs#L66), so implementation should be easy.

**Reply by sayanarijit** [reply]
Let's do it in https://github.com/sayanarijit/xplr/issues/448

**Comment by sayanarijit** [comment]
Done in https://github.com/sayanarijit/xplr/pull/450. Will be released soon.
